packages/fx-core/src/common/spec-parser/specParser.ts

```diff
@@ -192,13 +195,19 @@ export class SpecParser {
 
     await fs.writeJSON(manifestPath, updatedManifest, { spaces: 2 });
 
-    // TODO: other implementations
+    for (const url in newSpec.paths) {
+      const getOperation = newSpec.paths[url]?.get;
+      const card: AdaptiveCard = generateAdaptiveCard(getOperation!);
+      const fileName = path.join(adaptiveCardFolder, `${getOperation!.operationId!}.json`);
```

Comment:
just want to confirm about this. Looks like you will assume this folder always exists before this method was called?

Reply:
I am using fs.outputJSON, this will generate folder if it does not exist: https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputJson.md#outputjsonfile-object-options-callback